parts/intro.md

```diff
@@ -1,10 +1,22 @@
 # Introduction
 
 ## 1. What makes Python uniquely capable of handling NGS data?
-*Even to people outside science, I think this could be an interesting topic.*
+
+Python is a very versatile and easy to use language. It is famous for having a very
+quick learning curve:
+
+![Python XKCD](/assets/python.png?raw=True)
```

Comment:
I think we should put this XKCD in a slide :stuck_out_tongue: , if its readable of course